src/libs/folder.js

```diff
@@ -0,0 +1,205 @@
+import { models } from 'cozy-client'
```

Comment:
Is that the temporary code you mentionned this morning and that should be moved in cozy-client later? If yes we should mention this somewhere in the commit message or in the file comments. I no, maybe we should rename the file as `folder.js` is a bit too generic

Reply:
Thanks. I added a header to this file which explains the situation